MaterialLib/PhysicalConstant.h

```diff
@@ -74,6 +74,8 @@ const double O2 = 0.032;  ///< kg \per{mol}
  * - Source: http://www.engineeringtoolbox.com/molecular-mass-air-d_679.html
  */
 const double Air = 0.02897;  ///< kg \per{mol}
+
+const double H2 = 0.002;
```

Comment:
From https://pubchem.ncbi.nlm.nih.gov/compound/Hydrogen it is 2.016 g/mol.
What is your reference?